lib/FastImageSize.php

```diff
@@ -225,4 +208,58 @@ protected function getReturnData()
 	{
 		return sizeof($this->size) > 1 ? $this->size : false;
 	}
+
+	/**
+	 * Load all supported types
+	 */
+	protected function loadAllTypes()
+	{
+		foreach ($this->supportedTypes as $imageType => $extension)
+		{
+			$this->loadType($imageType, $extension);
```

Comment:
`$extension` is a superfluous argument, `loadType()` only has one parameter.

Reply:
Right, that was from a previous iteration.